packages/client/src/comp/AddressForm.tsx

```diff
@@ -41,54 +34,59 @@ const FlexFixed = styled.div`
 // pulled out for testing
 export const RawAddressForm: React.FC<{rawState: string, zip?: string}> = ({rawState, zip}) => {
   const { path, pushState } = useAppHistory()
-  const addrRef = useControlRef<Input>()
-  const { address, setAddress } = AddressContainer.useContainer()
+  const streetAddressRef = useControlRef<Input>()
+  const { setAddress } = AddressContainer.useContainer()
   const { setContact } = ContactContainer.useContainer()
   const { fetchingData, setFetchingData } = FetchingDataContainer.useContainer()
-
+  const {
```

Comment:
You need to create your own unstated container.  Factor out the parts you want to save from `FieldsContainer` so as to not duplicate code.